src/Xamarin.Android.Build.Tasks/Utilities/XAJavaTypeScanner.cs

```diff
@@ -48,7 +49,9 @@ public List<TypeDefinition> GetJavaTypes (ICollection<ITaskItem> inputAssemblies
 				throw new InvalidOperationException ($"Internal error: assembly '{asmItem.ItemSpec}' should be in the '{targetArch}' architecture, but is in '{arch}' instead.");
 			}
 
-			AssemblyDefinition asmdef = resolver.Load (asmItem.ItemSpec);
+			AssemblyDefinition? asmdef = resolver.Load (asmItem.ItemSpec);
+			if (asmdef == null)
+				continue;
```

Comment:
I think it would be nice if it logged a warning here.  This is not a normal situation, and skipping an assembly might lead to "mysterious" errors later on, if that assembly happens to have JLO types.

Reply:
I put `LogDebugMessage()`, as a warning would not be actionable. There wouldn't be a good way for a developer to fix the warning.